src/test/java/com/example/rets_api/repository/UtilsTest.java

```diff
@@ -35,16 +35,20 @@ public static PropertyEntity createPropertyEntityWithBasicFields(){
         propertyEntity.setPropertyTypeCondo(false);
         propertyEntity.setPropertyTypeTownHouse(true);
         propertyEntity.setFinancialData(createFinancialDataEntity());
-        propertyEntity.setRoomList(asList(createRoomEntity(RoomType.LIVING_ROOM)));
+        propertyEntity.setRoomList(asList(createRoomEntity(RoomType.LIVING_ROOM), createRoomEntity((RoomType.MAIN_FLOOR_BEDROOM))));
         propertyEntity.setSchoolList(asList(createSchoolEntity("primary", "jrHigh")));
         propertyEntity.setAnimalPolicy(createAnimalPolicyEntity());
         propertyEntity.setLotData(createLotDataEntity());
+        propertyEntity.setViewData(createViewDataEntity());
         return propertyEntity;
     }
 
     public static PropertyEntity createPropertyEntityResponseAndID(Long id){
         PropertyEntity propertyEntity = createPropertyEntityWithBasicFields();
         propertyEntity.setPropertyId(id);
+        propertyEntity.setViewData(createViewDataEntity());
+        propertyEntity.setLotData(UtilsTest.createLotDataEntity());
```

Comment:
you can remove setViewData(createViewDataEntity()) and setLotData(UtilsTest.createLotDataEntity()) because those values are being filled in createPropertyEntityWithBasicFields with the same values